fix: Could not parse file "org.kde.kguitar.desktop":
Key file does not have group ?Desktop Entry?

Index: kguitar_shell/org.kde.kguitar.desktop
--- kguitar_shell/org.kde.kguitar.desktop.orig
+++ kguitar_shell/org.kde.kguitar.desktop
@@ -1,4 +1,4 @@
-[KDE Desktop Entry]
+[Desktop Entry]
 Encoding=UTF-8
 Name=KGuitar
 Exec=kguitar %u -qwindowtitle "%c" %m
